package.json

```diff
@@ -103,10 +103,9 @@
     "github-release": "./node_modules/.bin/conventional-github-releaser",
     "changelog": "./node_modules/.bin/conventional-changelog -i CHANGELOG.md --same-file",
     "release": "yarn run clean && yarn run lint && yarn run test && yarn run prod",
-    "publish": "./build/publish.sh",
-    "minor": "./build/release.sh -n && yarn run publish",
-    "major": "./build/release.sh -m && yarn run publish",
-    "patch": "./build/release.sh -p && yarn run publish"
+    "minor": "./build/release.sh -n && ./build/publish.sh",
```

Comment:
Does this fix the duplication?

Reply:
Yea I think so. Still not entirely sure why but the script doesn't run twice if i directly run it